content/analyze/visualization-components/visualization-components.md

```diff
@@ -10,3 +10,15 @@ linked:
     - ./igv.md
 ---
 
+
+## Visualization Packages
+The HCA Visualization Package Registry lists JavaScript modules that enable analysis through interactive data exploration on the web.  
+
+ These solutions are not built by the HCA DCP implementation team but are solutions we foster and support. 
+
+Portal developers can find instructions for integrating these visualizations into a web app on each component's detail page. Sample code showing a minimal reference implementation is available for some components by clicking on their "mashup" badge.
```

Comment:
Let's delete or (preferably, if possible) comment out this part about detail pages until they're ready.

```suggestion
```